<ui version="4.0" >
 <class>iDirSearch</class>
 <widget class="QDialog" name="iDirSearch" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>230</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Additional  Directories</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>331</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="m_pNewLine" >
         <property name="toolTip" >
          <string>New line</string>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset>../../../../../CodeAnalyst/CA_Win/source/profiler/gui/images/addtab.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pRemoveLine" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="toolTip" >
          <string>Remove line</string>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset>../../../../../CodeAnalyst/CA_Win/source/profiler/gui/images/closetab.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pUp" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="toolTip" >
          <string>Line up</string>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset>../../../../../CodeAnalyst/CA_Win/source/profiler/gui/images/up.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pDown" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="toolTip" >
          <string>Line down</string>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset>../../../../../CodeAnalyst/CA_Win/source/profiler/gui/images/down.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QTableWidget" name="m_pDirTable" >
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="showGrid" >
      <bool>false</bool>
     </property>
     <property name="columnCount" >
      <number>1</number>
     </property>
     <column>
      <property name="text" >
       <string>Directories</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>361</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="m_pOk" >
         <property name="text" >
          <string>Ok</string>
         </property>
         <property name="shortcut" >
          <string>Alt+O</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pCancel" >
         <property name="text" >
          <string>Cancel</string>
         </property>
         <property name="shortcut" >
          <string>Alt+C</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>m_pOk</sender>
   <signal>clicked()</signal>
   <receiver>iDirSearch</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pCancel</sender>
   <signal>clicked()</signal>
   <receiver>iDirSearch</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
